torch/onnx/symbolic_opset11.py

```diff
@@ -249,7 +249,12 @@ def _len(g, self):
 
 
 def __getitem_(g, self, i):
-    return g.op("SequenceAt", self, i)
+    if self.type().isSubtypeOf(torch._C.ListType.ofTensors()):
+        # SequenceAt requires that the input be a List of Tensors
+        return g.op("SequenceAt", self, i)
+    else:
+        from torch.onnx.symbolic_opset9 import select
+        return select(g, self, g.op("Constant", value_t=torch.tensor([0])), i)
```

Comment:
nit: can we import and use `__getitem_` in opset 9?

Reply:
done